Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -2647,3 +2647,106 @@ function Remove-CustomRole {
     }
 }
 
+Function Get-vSANDataInTransitEncryptionStatus {
+    <#
+    .DESCRIPTION
+        Gets status of vSAN Data-In-Transit Encryption for all clusters in a SDDC
+    #>
+    [CmdletBinding()]
+    [AVSAttribute(10, UpdatesSDDC = $false)]
+    param()
```

Comment:
Would it make sense to take ClusterName as a parameter?

Reply:
So, why not add it to the get-function signature as well?

Reply:
That's not really required - because it would help show status of all the clusters.